server/channels/app/options.go

```diff
@@ -110,8 +110,10 @@ func SkipPostInitialization() Option {
 	}
 }
 
-type AppOption func(a *App)
-type AppOptionCreator func() []AppOption
+type (
+	AppOption        func(a *App)
+	AppOptionCreator func() []AppOption
+)
```

Comment:
1/5 is quite uncommon to use `type` blocks. The pervious pattern is fine

Reply:
This is reformated automatically by my editor, I don't know exactly why, let me double check.